cypress/TestCases/FireboltCertification/UserInterest.feature

```diff
@@ -0,0 +1,130 @@
+            | interest & reason recording with program entity                                        | type interest and reason recording with program entity                                        | type interest reason recording program entity                                        |
+            | disinterest & reason recording with program entity                                     | type disinterest and reason recording with program entity                                     | type disinterest reason recording program entity                                     |
+            | interest & reason recording with channel streaming entity                              | type interest and reason recording with channel streaming entity                              | type interest reason recording channel streaming entity                              |
+            | disinterest & reason recording with channel streaming entity                           | type disinterest and reason recording with channel streaming entity                           | type disinterest reason recording channel streaming entity                           |
+            | interest & reason recording with channel overTheAir entity                             | type interest and reason recording with channel overTheAir entity                             | type interest reason recording channel overTheAir entity                             |
+            | disinterest & reason recording with channel overTheAir entity                          | type disinterest and reason recording with channel overTheAir entity                          | type disinterest reason recording channel overTheAir entity                          |
+            | interest & reason recording with program episode entity                                | type interest and reason recording with program episode entity                                | type interest reason recording program episode entity                                |
+            | disinterest & reason recording with program episode entity                             | type disinterest and reason recording with program episode entity                             | type disinterest reason recording program episode entity                             |
+            | interest & reason recording with program season entity                                 | type interest and reason recording with program season entity                                 | type interest reason recording program season entity                                 |
+            | disinterest & reason recording with program season entity                              | type disinterest and reason recording with program season entity                              | type disinterest reason recording program season entity                              |
+
+
```

Comment:
Review comment on line 63 is not addressed. userInterest is not a get call. It can be changed to notify

Reply:
corrected